Tests/FluentAssertions.Json.Specs/ShouldBeJsonSerializableTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using AutoFixture;
+using FluentAssertions;
+using FluentAssertions.Json.Specs.Models;
+using Xunit;
+
+// NOTE that we are using both namespaces 'FluentAssertions' & 'FluentAssertions.Json' from an external namespace to force compiler disambiguation warnings
+// ReSharper disable CheckNamespace
+namespace SomeOtherNamespace
+// ReSharper restore CheckNamespace
+{
+    public class ShouldBeJsonSerializableTests
+    {
+        private readonly Fixture _fixture;
+
+        public ShouldBeJsonSerializableTests()
+        {
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public void Class_is_simple_poco_should_be_serializable()
```

Comment:
🤔 I don't quite get this naming convention in all of these test cases. For this one, did you intend to say "Class as simple poco should be serializable"? Maybe "Simple_poco_should_be_serializable" or "When_class_is_simple_poco_it_should_be_serializable"?